addons/delivery/models/stock_picking.py

```diff
@@ -92,6 +92,14 @@ def _compute_shipping_weight(self):
     package_ids = fields.Many2many('stock.quant.package', compute='_compute_packages', string='Packages')
     weight_bulk = fields.Float('Bulk Weight', compute='_compute_bulk_weight')
     shipping_weight = fields.Float("Weight for Shipping", compute='_compute_shipping_weight')
+    carrier_tracking_url = fields.Char(string='Tracking Url', compute='_compute_carrier_tracking_url', store=True)
+    is_sent_delivery_mail = fields.Boolean()
```

Comment:
I still don't like this field name. What do you think of "mail_sent" ? or "delivery_mail_sent" ? Do you have any suggestion ?